src/store/reducers/appReducer.js

```diff
@@ -46,6 +63,24 @@ export const epics = createEpicScenario({
         .catch(handleError(fromAction))
   },
 
+  /** Get solution settings */
+  getSolutionSettings: {
+    type: 'APP_SOLUTION_GET_SETTINGS',
+    epic: () =>
+      ConfigService.getSolutionSettings()
+        .map(redux.actions.updateSolutionSettings)
+    // TODO handle error
+  },
+
+  /** Update solution settings */
+  updateSolutionSettings: {
+    type: 'APP_SOLUTION_UPDATE_SETTINGS',
+    epic: ({ payload }) =>
+      ConfigService.updateSolutionSettings(payload)
+        .map(redux.actions.updateSolutionSettings)
+    // TODO handle error
```

Comment:
same here

Reply:
This will only add the error to state. It will not show the error to the user